pkg/faketikv/event.go

```diff
@@ -75,9 +79,9 @@ type WriteFlowOnSpot struct {
 }
 
 // Run implements the event interface
-func (w *WriteFlowOnSpot) Run(driver *Driver) bool {
-	raft := driver.raftEngine
-	res := w.in.Step(driver.tickCount)
+func (w *WriteFlowOnSpot) Run(er *EventRunner, tickCount int64) bool {
```

Comment:
`EventRunner` run each event. It's strange that event should know `EventRunner`.